app/Http/Resources/V1/SearchResource.php

```diff
@@ -18,7 +18,7 @@ public function toArray($request)
     {
         return [
             'id' => $this->id,
-            'thumb_url' => $this->when(isset($this->thumb_url), $this->thumb_url),
+            'thumb_url' => (isset($this->thumb_url) ? $this->thumb_url : $this->thumbUrl),
```

Comment:
When does $this->thumbUrl exist?

Reply:
It exists if the resource is Playlists.

Basically "thumb_url" attribute exists for Projects and Activities,
but Playlists do not have "thumb_url" attribute.
Leo has asked to use Projects "thumb_url" attribute for Playlists.
"thumbUrl" is an accessor Playlists model.

Reply:
I can't find that usage. Can you point it out?

Reply:
> I can't find that usage. Can you point it out?

Sorry I don't get your point which usage you can not find.